src/MaaCore/Task/Miscellaneous/StartGameTaskPlugin.cpp

```diff
@@ -9,7 +9,15 @@ bool StartGameTaskPlugin::_run()
     if (m_client_type.empty()) {
         return false;
     }
-    return ctrler()->start_game(m_client_type);
+
+    for (int i = 0; i < 4; i++) {
+        if (!ctrler()->start_game(m_client_type)) {
+            return false;
+        }
+        sleep(2000);
+    }
```

Comment:
```suggestion
    do {
        if (!ctrler()->start_game(m_client_type)) {
            return false;
        }
        sleep(2000);

    } while (pipe_data.size() < 168);
```

Reply:
Ideally if we could use pipe_data.size() in StartGameTaskPlugin.cpp, this should work